src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -36,6 +36,14 @@ def __init__(
             feature_extractor=feature_extractor,
         )
 
+    def set_attention_chunk(self, chunk_size: Optional[Union[str, int]] = "auto"):
+        # set chunk_size = `None` to disable `set_attention_chunk`
+        if chunk_size == "auto":
+            # half the attention head size is usually a good trade-off between
+            # speed and memory
+            chunk_size = self.unet.config.attention_head_dim // 2
```

Comment:
think this should be handled in the model.

Reply:
Don't agree - this is a good default for stable diffusion, not for general UNets